la_palma_overview/la_palma_overview_video.py

```diff
@@ -86,14 +96,14 @@ def make_video_from_images(path2night, video_path):
     """
     avconv_command = [
         'avconv',
-        '-y', # force overwriting of existing output file 
-        '-framerate', '12', # 12 Frames per second
-        '-f', 'image2', 
-        '-i', os.path.join(path2night,'%06d.jpg'), 
+        '-y',  # force overwriting of existing output file
+        '-framerate', '12',  # 12 Frames per second
+        '-f', 'image2',
+        '-i',  os.path.join(path2night, '%06d.jpg'),
         '-c:v', 'h264',
-        '-s', '1920x1080', # sample images down to FullHD 1080p
-        '-crf', '23', # high quality 0 (best) to 53 (worst)
-        '-crf_max', '25', # worst quality allowed
+        '-s', '1920x1080',  # sample images down to FullHD 1080p
+        '-crf', '23',  # high quality 0 (best) to 53 (worst)
+        '-crf_max', '25',  # worst quality allowed
```

Comment:
instead of writing out these cmdline arguments for avconv here one can use `shlex.split` on a string ... possibly a multiline string.